modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/CodeEditor/CodeEditorLocalized.tsx

```diff
@@ -0,0 +1,186 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import ClayDropDown from '@clayui/drop-down';
+import ClayIcon from '@clayui/icon';
+import ClayLabel from '@clayui/label';
+import ClayLayout from '@clayui/layout';
+import ClayLoadingIndicator from '@clayui/loading-indicator';
+import React, {ReactNode, useEffect, useState} from 'react';
+
+import {SidebarCategory} from './Sidebar';
+import CodeEditor from './index';
+
+import './CodeEditorLocalized.scss';
+
+interface CodeEditorLocalizedProps {
+	CustomSidebarContent?: ReactNode;
+	ariaLabels?: {
+		default: string;
+		openLocalizations: string;
+		translated: string;
+		untranslated: string;
+	};
+	mode?: string;
+	onSelectedLocaleChange: (val: IItem) => void;
+	onTranslationsChange: (val: LocalizedValue<string>) => void;
+	placeholder?: string;
+	selectedLocale: Locale;
+	sidebarElements: SidebarCategory[];
+	translations: LocalizedValue<string>;
+}
+
+interface IItem {
+	label: Locale;
+	symbol: string;
+}
+
+const defaultLanguageId = Liferay.ThemeDisplay.getDefaultLanguageId();
+
+const availableLocales = Object.keys(Liferay.Language.available)
+	.sort((languageId) => (languageId === defaultLanguageId ? -1 : 1))
+	.map((language) => ({
+		label: language as Locale,
+		symbol: language.replace('_', '-').toLowerCase(),
+	}));
+
+export function CodeEditorLocalized({
+	CustomSidebarContent,
+	ariaLabels = {
+		default: 'Default',
+		openLocalizations: 'Open Localizations',
+		translated: 'Translated',
+		untranslated: 'Untranslated',
+	},
```

Comment:
Why not translating ariaLabels? Like:
`ariaLabels = {default: Liferay.language.get('default') ...},`

Reply:
Thanks for noticing that @cgmonte, I already made your suggestion :smile: